### PR TITLE
Project Setup

### PR DESCRIPTION
**Pull request by ZeenatLawal**
- Setup project with webpack
- Add HTML Markup
- Style HTML Markup
- Add function to display scores
- Integrate linters

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,7 +1,70 @@
...
+
+.scores {
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+.card {
+  width: 45%;
+}
```
Comment (henatan99) [inline src/style.css:63]: - [ ] I think it is better if you keep the horizontal alignment of your card class components to stick with the original wireframe design to fulfill the minimum styling. 
image
image

File: src/style.css [hunk 2]
```diff
@@ -1,7 +1,70 @@
...
+
 body {
-  background-color: bisque;
+  background-color: white;
+}
+
+h1 {
+  padding: 10px;
+  text-align: center;
```
Comment (henatan99) [inline src/style.css]: - In the original wireframe design, the title is aligned left. Kindly stick with the original design to have the minimum styling. 
image
image

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @ZeenatLawal >, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 2 above)
Comment (rloterh) [reply src/style.css]: This issue is well resolved! ✔️

File: src/style.css (hunk 1 above)
Comment (rloterh) [reply src/style.css:63]: This issue is not properly resolved.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Zeena, 

You did great on this project, and for making the changes requested by the previous reviewer!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 

## Good Points 👍  
- No linter errors

## Required Changes ♻️
- [ ] _Kindly check the comments above, made under the reviews of the previous code reviewer(s) and stating **not properly resolved**. Kindly take your time to correctly review those issues outlined by the previous code reviewers, and please check to see that the changes are working as intended._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ZeenatLawal** [comment]
Hi @rloterh, I'm not sure this review is for my project because the name stated is Paka and Yannick-Noel. Can you please clarify?

**Comment by ZeenatLawal** [comment]
Hi @rloterh, I'm not sure this review is for my project because the name stated is Paka and Yannick-Noel. Can you please clarify?

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @ZeenatLawal, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Dipeshtwis** [comment]
> Hi @rloterh, I'm not sure this review is for my project because the name stated is Paka and Yannick-Noel. Can you please clarify?

Maybe he mistakenly put that name, You don't have to worry about this.
Thank you

File: src/style.css (hunk 1 above)
Comment (Dipeshtwis) [reply src/style.css:63]: Well addressed!! :clap:

**Comment by ZeenatLawal** [comment]
@Dipeshtwis I thought so too, that's why I went ahead and implemented the changes. Thank you

**Comment by rloterh** [comment]
> Hi @rloterh, I'm not sure this review is for my project because the name stated is Paka and Yannick-Noel. Can you please clarify?

Yes, as indicated by Dipesh, there was a mistake in the naming, but the review is in order. I'll make the necessary corrections

**Pull request merged**